test/tools/router_check/router.cc

```diff
@@ -0,0 +1,204 @@
+#include "test/tools/router_check/router.h"
+
+Json::ObjectPtr RouterCheckTool::loadJson(const std::string& config_json,
+                                          const std::string& schema) {
+
+  try {
+    // Json configuration
```

Comment:
general coding style preference: There are a lot of comments in this change that are obvious. For example if you have something like this:

// do something cool
doSomethingCool(); 

There is no point in having a comment. Can you go through and delete comments that fall into that category (there are many).